diff.py

```diff
@@ -984,6 +993,9 @@ def maybe_get_objdump_source_flags(config: Config) -> List[str]:
         if config.inlines:
             flags.append("--inlines")
 
+    if config.diff_section is not None:
+        flags.append("-j" + config.diff_section)
```

Comment:
this shouldn't be needed, both callers of `maybe_get_objdump_source_flags` already pass this -j flag